mmedit/datasets/pipelines/augmentation.py

```diff
@@ -1031,3 +1031,86 @@ def __repr__(self):
         repr_str = self.__class__.__name__
         repr_str += (f'(keys={self.keys})')
         return repr_str
+
+
+@PIPELINES.register_module()
+class CopyValueFromKey:
+    """Copy the value of a source key to a destination key.
+
+    It does the following: results[dst_key] = results[src_key].
+
+    Args:
+        src_key (str): The source key.
```

Comment:
Do you need to support specifying a list of keys in one go?

Reply:
No for now, but let us support that now in case we need that in the future.